### PR TITLE
Little cleaning...

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/framework/audio/internal/plugins/knownaudiopluginsregister.h [hunk 1]
```diff
@@ -25,15 +25,18 @@
 
 #include "audio/iknownaudiopluginsregister.h"
```
Comment (Eism) [inline src/framework/audio/internal/plugins/knownaudiopluginsregister.h:26]: I would suggest moving "audio/iknownaudiopluginsregister.h" to the very bottom and without "audio"

File: src/framework/audio/tests/knownaudiopluginsregistertest.cpp [hunk 2]
```diff
@@ -22,11 +22,11 @@
...
 #include <gtest/gtest.h>
 
-#include "audio/internal/plugins/knownaudiopluginsregister.h"
+#include "internal/plugins/knownaudiopluginsregister.h"
 #include "global/tests/mocks/filesystemmock.h"
-#include "audio/tests/mocks/audioconfigurationmock.h"
+#include "tests/mocks/audioconfigurationmock.h"
 
-#include "serialization/json.h"
+#include "global/serialization/json.h"
```
Comment (Eism) [inline src/framework/audio/tests/knownaudiopluginsregistertest.cpp:29]: needs sorting

**Review by Eism — APPROVED** [review]

File: src/framework/audio/internal/plugins/knownaudiopluginsregister.h (hunk 1 above)
Comment (igorkorsukov) [reply src/framework/audio/internal/plugins/knownaudiopluginsregister.h]: fixed

File: src/framework/audio/tests/knownaudiopluginsregistertest.cpp (hunk 2 above)
Comment (igorkorsukov) [reply src/framework/audio/tests/knownaudiopluginsregistertest.cpp]: fixed

**Pull request merged**